src/controller/providers/providers.controller.ts

```diff
@@ -0,0 +1,26 @@
+import { Controller, Get, Logger } from '@nestjs/common';
+import { ApiOkResponse, ApiOperation } from '@nestjs/swagger';
+import { StorageProviderService } from 'src/service/storage-provider/storage-provider.service';
+import { StorageProviderWithIpInfo } from 'src/service/storage-provider/types.storage-provider';
+
+@Controller('providers')
+export class ProvidersController {
```

Comment:
`StorageProvidersController`, also I'd set filename for `/storage-providers/storage-providers.controller.ts`

Reply:
Done. Had to rename `StorageProvidersController` from `stats` directory to `StorageProvidersStatsController`.